src/visitor/sqlite.rs

```diff
@@ -201,8 +201,10 @@ impl<'a> Visitor<'a> for Sqlite<'a> {
                 self.write(" RETURNING ")?;
 
                 for (i, column) in returning.into_iter().enumerate() {
+                    //yay https://sqlite.org/forum/info/6c141f151fa5c444db257eb4d95c302b70bfe5515901cf987e83ed8ebd434c49?t=h
+                    self.delimited_identifiers(&[&*column.name])?;
```

Comment:
why do we use `delimited_identifiers` for a single identifier here?

Reply:
Good point, refactored it.